docs/lang/articles/math/math_module.md

```diff
@@ -104,23 +101,24 @@ u = v.wzyx  # vec4(4, 3, 2, 1)
 u = v.rraa  # vec4(1, 1, 2, 2)
 ```
 
-### Relationship between `ti.Vector`, `ti.types.vector` and `ti.math.vec3`
+### Differentiate `ti.Vector`, `ti.types.vector` and `ti.math.vec3`
 
 + `ti.Vector` is a function that accepts a 1D array and returns a matrix instance that has only one column. For example, `ti.Vector([1, 2, 3, 4, 5])`.
-+ `ti.types.vector` is a function that accepts an integer and a primitive type, and returns a vector type. For example: `vec5f = ti.types.vector(5, float)`. `vec5f` can then be used to instantiate 5D vectors or annotate data types in function arguments and struct members:
++ `ti.types.vector` is a function that accepts an integer and a primitive type and returns a vector type. For example: `vec5f = ti.types.vector(5, float)`. `vec5f` can then be used to instantiate 5D vectors or annotate data types of function arguments and struct members:
+
     ```python
     @ti.kernel
     def test(v: vec5f):
         print(v.xyz)
     ```
-    Unlike `ti.Vector`, whose input data must be a 1D array, vector types created by `ti.types.vector()` have more flexible ways to initialize, as explained above.
-+ `ti.math.vec3` is simply created by `vec3 = ti.types.vector(3, float)`.
+    Unlike `ti.Vector`, whose input data must be a 1D array, `ti.types.vector()` creates vector types that allow more flexible initiation rules, as explained above.
++ `ti.math.vec3` is created by `vec3 = ti.types.vector(3, float)`.
 
 
 ## GLSL-standard functions
 
 
-Taichi's math module also supports a few [GLSL standard functions](https://registry.khronos.org/OpenGL-Refpages/gl4/index.php). These functions follow the GLSL standard, except that they accept arbitrary vectors and matrices as arguments and operate on them element-wise. For example:
+Taichi's math module also supports a few [GLSL standard functions](https://registry.khronos.org/OpenGL-Refpages/gl4/index.php). These functions follow the GLSL standard, except that they accept arbitrary vectors and matrices as arguments and perform element-wise operations. For example:
```

Comment:
The original is good. Could I ask you to use the original?
Your version is unclear because it didn't specify the target of these operations: The matrices and vectors.